mbed/mbed.py

```diff
@@ -2780,6 +2792,26 @@ def toolchain_(name=None, global_cfg=False, supported=False):
         return compile_(supported=supported)
     return config_('toolchain', name, global_cfg=global_cfg)
 
+@subcommand('profile',
```

Comment:
Looks like duplicate code, which might be result of the rebase issue I mentioned above. Do you think you could remove the subcommand for now until there's more clarity on `mbed <verbs>`